test/integration/targets/docker_container/tasks/tests/options.yml

```diff
@@ -2773,6 +2773,53 @@
     - pid_mode_3 is changed
   when: docker_py_version is version('2.0.0', '<')
 
+####################################################################
+## pids_limit ######################################################
+####################################################################
+
+- name: pids_limit
+  docker_container:
+    image: alpine:3.8
+    command: '/bin/sh -c "sleep 10m"'
+    name: "{{ cname }}"
+    state: started
+    pids_limit: 10
+    force_kill: yes
+  register: pids_limit_1
```

Comment:
Can you adjust the tests simliar to the ones for `dns_opts`, so that they also run through for docker-py < 1.10.0?